app/models/visit.rb

```diff
@@ -18,7 +18,10 @@ class Visit
   }
   validate :validate_amount_of_adults, on: :visitors_set
 
-  delegate :prison_name, :prison_name=, to: :prisoner, allow_nil: true
+  delegate :prison, :prison_name, :prison_name=,
+    to: :prisoner, allow_nil: true
+  delegate :prison_email, :prison_canned_responses, :prison_nomis_id,
+    to: :prisoner
 
   def validate_amount_of_adults
     if visitors.none? { |v| v.age && v.age >= 18 }
```

Comment:
Actually it should be using its own method for this, I think...Although I think AgeValidator might be a candidate for deprecation now as it's only used to add a single, albeit complex, validation to Visit.   Can't see any reason why it needs to be it's own class any longer.